### ISSUE TITLE
Support of custom skill

### ISSUE THREAD

**Issue by binford6000** [fhem/alexa-fhem-docker#163]
Recently i tried to move away from local installation towards docker deployment. Everything worked fine despite the connection to my custom skill [like reported here](https://forum.fhem.de/index.php?topic=135435.msg1290228#msg1290228).

The question is if custom skill is supported? Theoretically it should while its just another section in the config.json. But it might
doesn't connect to the fhemIntents attribute over at fhem where the custom skill functions are listed.

Errors are:
`ERROR: InvalidAccessTokenError from ::ffff:34.245.84.101`
and
`clientID amzn1.application-oa2-client.xxxxxxxxxxxxxxxxxxxxxxxxxxx not authorized`

Any suggestion?
Cheers,
Sebastian

**Comment by sidey79** [comment]
I think it should work.

Did you Open the Port for your custom Skill?

Whats in the logs of the Alexa docker container?

**Comment by binford6000** [comment]
> Did you Open the Port for your custom Skill?

Yes, i did  open port 8002 like i did in the original setup. I put the port in the docker-compose.yml file as well.
> Whats in the logs of the Alexa docker container?

`ERROR: InvalidAccessTokenError from ::ffff:34.245.84.101`

**Comment by sidey79** [comment]
I think this is more a configuration issue of alexa-fhem as a issue of the container.

The message tells you that the access token isn't accepted, so there seems to be a problem with the token which ist provided to alexa-fhem.

May a lost char during your copy command?
I never implemented the custom skill myself, so i am not exactly sure where this token is generated and where it must be stored.

**Comment by binford6000** [comment]
> I think this is more a configuration issue of alexa-fhem as a issue of the container.

Yeah could be the case. But trust me, i double checked it before i posted that issue on the FHEM forum. 

> The message tells you that the access token isn't accepted, so there seems to be a problem with the token which ist provided to alexa-fhem.

This seems to be obvious. 

> May a lost char during your copy command?
I never implemented the custom skill myself, so i am not exactly sure where this token is generated and where it must be stored.

Both the app id and the token is generated on the alexa side. It has to be put in the config.json like this:
`"applicationId": "amzn1.ask.skill.xxxxxxxxxxxxxxxxxxxxxxxxxxxx",`
`"oauthClientID": "amzn1.application-oa2-client.xxxxxxxxxxxxxxxxxxxxxxxxxx"`

Anyways - when i will have some spare time i will give it try once more. I will record all steps to paint the full picture.
In the best case it is really a c&p issue. But i don't tink so tbh...
